README.md

```diff
@@ -118,6 +118,11 @@ gProfiler can be run in a continuous mode, profiling periodically, using the `--
 Note that when using `--continuous` with `--output-dir`, a new file will be created during *each* sampling interval.
 Aggregations are only available when uploading to the Granulate Performance Studio.
 
+### Uploading profiling data from another source
+gProfiler can be used to upload external .col file to Granulate Performance Studio by using `upload-file` subcommand.
+In case of gProfiler run volume mapping flag must be added. Example: `docker run --name granulate-gprofiler -v <path-to-.col>:<path-to-.col> --pid=host --userns=host --privileged  gprofiler:latest upload-file  --token=<token> --service-name="<service>" --file-path <path-to-.col>`
```

Comment:
Mention that it's only for docker mode (maybe you meant "in case of docker run, ..."?)